src/Features/LanguageServer/Protocol/Handler/InlayHint/InlayHintHandler.cs

```diff
@@ -88,10 +88,10 @@ public TextDocumentIdentifier GetTextDocumentIdentifier(InlayHintParams request)
                     Data = new InlayHintResolveData(resultId, i, request.TextDocument)
                 };
 
-                inlayHints.Add(inlayHint);
+                inlayHints[0] = inlayHint;
```

Comment:
>[0] [](http://example.com/codeflow?start=26&length=3)

[i]

Reply:
do you think it would matter in practice?  0 seems liek a reasonable approximation.

Reply:
Very true, especially for some values of i